data/PA/incentives.json

```diff
@@ -1089,5 +1089,67 @@
     "start_date": "2023-07-01",
     "end_date": "2024-06-30",
     "low_income": "pa-pennsylvania-department-of-environmental-protection-bonus"
+  },
+  {
+    "id": "PA-51",
+    "authority_type": "utility",
+    "authority": "pa-ugi-utilities",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "heat_pump_air_conditioner_heater",
+    "program": "pa_uGIUtilities_equipmentRebateProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 1250,
+      "minimum": 150,
+      "maximum": 1250
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "Up to $1,250 rebate for ENERGY STAR® certified mini-split heat pumps. Rebate amount varies depending on heat pump efficiency and capacity."
+    }
+  },
+  {
+    "id": "PA-52",
+    "authority_type": "utility",
+    "authority": "pa-ugi-utilities",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "smart_thermostat",
+    "program": "pa_uGIUtilities_applianceRebateProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 50
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$50 rebate for ENERGY STAR® certified smart thermostats."
+    }
+  },
+  {
+    "id": "PA-53",
+    "authority_type": "utility",
+    "authority": "pa-peco",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "new_electric_vehicle",
```

Comment:
Does this apply to used EVs too?

If so, I'm actually not sure we should duplicate the incentive with `item: used_electric_vehicle`. In our calculator, we always show new and used EV incentives together, so not having the duplicate will actually make for a better user experience. We can (i.e. I can) add `used_electric_vehicle` when we (I) let incentives have multiple items.

In any case, could you reflect the new/used situation in the description? E.g. "$50 rebate for purchasing a new or used EV".

Reply:
their website doesn't specify and there's no way to get more details without logging into a PECO account, so I just assumed it was new to avoid misleading anyone